### PR TITLE
Debate outcomes

### PR DESCRIPTION
**Pull request by h-lame**
For: https://www.pivotaltracker.com/story/show/96654388

Add a debate outcome model attached (has_one) to petition.  Allow admins to fill in the details of this model via a new form.  Publish that information - if available - on the petition show in the right place (under the 100,000 signatures info block).

### CODE REVIEW

File: app/views/admin/debate_outcomes/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<%= form_for @petition.debate_outcome, url: admin_petition_debate_outcome_path(@petition), method: :patch do |f| -%>
+
+  <%= render 'admin/petitions/petition_details', f: f %>
+
+  <h2>Debate Outcome</h2>
+
+  <%= form_row :for => [f.object, :debated_on] do %>
+    <%= f.label :debated_on, class: 'form-label' %>
+    <%= error_messages_for_field f.object, :debated_on %>
+    <%= f.text_field :debated_on, type: 'date', tabindex: increment, class: 'form-control' %>
```
Comment (otlaitil) [inline app/views/admin/debate_outcomes/show.html.erb]: You could use `date_field` instead? 

http://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-date_field

Comment (h-lame) [inline app/views/admin/debate_outcomes/show.html.erb]: Didn't know that was there.  Will do, thanks!

Comment (h-lame) [inline app/views/admin/debate_outcomes/show.html.erb]: Oh, and there's a ``url_field`` too for ``transcript_url`` and ``video_url``, neat. http://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-url_field

Comment (otlaitil) [inline app/views/admin/debate_outcomes/show.html.erb]: Cool!

**Comment by otlaitil** [comment]
Looks good! Nice to have a separate model for the outcomes. :+1:

**Comment by mybits** [comment]
Great! :+1:

**Comment by davidbasalla** [comment]
Looks like it needs a rebase?

**Pull request merged**